extensions/amp-script/0.1/amp-script.js

```diff
@@ -157,28 +167,14 @@ export class AmpScript extends AMP.BaseElement {
       );
     }
 
-    return getElementServiceForDoc(this.element, TAG, TAG).then((service) => {
-      this.setService(/** @type {!AmpScriptService} */ (service));
-    });
-  }
-
-  /**
-   * @override
-   */
-  onLayoutMeasure() {
-    if (this.layoutCompleted_ || this.reportedZeroSize_) {
-      return;
-    }
-
-    const {width, height} = this.getLayoutSize();
-    if (width === 0 && height === 0) {
-      this.reportedZeroSize_ = true;
-      user().warn(
-        TAG,
-        'Skipped initializing amp-script due to zero width and height.',
-        this.element
-      );
-    }
+    return getElementServiceForDoc(this.element, TAG, TAG)
+      .then((service) => {
+        this.setService(/** @type {!AmpScriptService} */ (service));
+      })
+      .then(() => {
+        this.setReadyState(ReadyState.LOADING);
+        this.layoutCallback();
```

Comment:
This is actually a biggish change, but can be made smaller if we do `return this.layoutCallback()`. But it's up to you - depends on how far you want to go.